tests/test.spec.ts

```diff
@@ -67,7 +67,7 @@ describe('templates can be instantiated', () => {
           datasourceProvider: 'mysql',
           repositoryOwner: 'prisma',
           repositoryHandle: 'templates-node',
-          engineType: 'library',
+          engineType: 'dataproxy',
```

Comment:
Add another test to make sure the version is correct when there is no `engineType` + `engineType === library/binary`, otherwise this test is "incomplete", and will not catch certain problems.

Reply:
Done

Reply:
It's fine if we set the engine in the schema to `library` or `binary` without updating the `@prisma/client` dependency right?

Reply:
Yep, library and binary are accepted engineTypes on stable

Reply:
But you should add snapshot tests for the Prisma schema as well, to make sure an engineType isn't added when not necessary. engineType is an escape hatch only.

Reply:
Okay done